confd/vars/docker-compose.yaml

```diff
@@ -9,9 +9,9 @@
 #no_elasticsearch: true
 #no_logstash: true
 #no_grpc_stub: true
-no_server42_server: true
```

Comment:
you remove all server42 instances stub and real dpdk

Reply:
It was done intentionally, I guess.
(for running real s42 on virtual env)